### DISCUSSION TITLE
failure: repodata/repomd.xml from docker-ce-stable: [Errno 256] No more mirrors to try

### DISCUSSION THREAD

**Discussion by horwedja** [Security-Onion-Solutions/securityonion#7026] [General] [answered]
Im stuck on settting up a sensor node. i get the below error:

http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repo                                                                                                                                                             data/repomd.xml: [Errno 14] curl#6 - "Could not resolve host: repocache.security                                                                                                                                                             onion.net; Unknown error"
  FAILURE
Pre-flight checks could not complete.

**Comment by dougburks** [comment, marked as answer]
Check to see if the sensor node is able to connect to apt-cacher-ng on the manager:
https://docs.securityonion.net/en/2.3/firewall.html#node-communication
